datajoint/user_relations.py

```diff
@@ -2,59 +2,83 @@
 Hosts the table tiers, user relations should be derived from.
 """
 
-import abc
 from .base_relation import BaseRelation
 from .autopopulate import AutoPopulate
 from .utils import from_camel_case
 from . import DataJointError
 
 
-class Part(BaseRelation, metaclass=abc.ABCMeta):
+class classproperty:
```

Comment:
Yes, but I think it is just last time the case for using them was not as compelling.  Let's review that again.  Frankly, I did not understand classproperties very well last time and it seemed like then we found an equally good solution without them.

In this case, introducing class properties substantially simplified the code in `erd` for looking up the `master` class.